src/nodejsWorker.ts

```diff
@@ -1,7 +1,7 @@
 var logPrefix = "LanguageWorkerConsoleLog";
 var errorPrefix = logPrefix + "[error] ";
 var ACTIVE_LTS_VERSION = "v8";
-var CURRENT_BRANCH_VERSION = "v10";
+var SUPPORTED_ACTIVE_BRANCH = "v10";

```

Comment:
`ACTIVE_LTS_VERSION_1` ?

Reply:
Yep. As an aside, we should probably collapse this and current_branch_version into an array of supported versions, once v12 comes out.

Reply:
I've gone ahead and changed that to an array of supported versions. Looks a bit cleaner.